apps/wakunode2/wakunode2.nim

```diff
@@ -215,7 +215,8 @@ proc retrieveDynamicBootstrapNodes*(dnsDiscovery: bool, dnsDiscoveryUrl: string,
 
 proc initNode(conf: WakuNodeConf,
               peerStore: Option[WakuPeerStorage],
-              dynamicBootstrapNodes: openArray[RemotePeerInfo] = @[]): SetupResult[WakuNode] =
+              dynamicBootstrapNodes: openArray[RemotePeerInfo] = @[],
+              rng: ref HmacDrbgContext): SetupResult[WakuNode] =
```

Comment:
As this parameter is mandatory, it should be located before the parameters with default values.

Doing that, you will avoid passing the RNG to this proc.

Reply:
Addressed in 1b9566c8